fedimint-bin-tests/src/main.rs

```diff
@@ -31,6 +31,8 @@ use tracing::{info, warn};
 mod util;
 use util::*;
 
+mod dirs;
```

Comment:
Why call it `dirs`?

Reply:
Because I chose the name before writing the code. Should be called env or vars